js/resources.js

```diff
@@ -0,0 +1,19 @@
+$(document).ready(function() {
+    $("#menu .filter .filter-items > a").click(function() { 
+        $("#menu .filter .filter-items > a").removeClass("active");  
+        $(this).toggleClass("active");
```

Comment:
should fix the removeClass("active") so it doesn't remove it from the current selected. This way the toggle in the next line would let you deselect the current filter.